core/src/main/java/tc/oc/pgm/score/ScoreMatchModule.java

```diff
@@ -89,6 +89,14 @@ public int getScoreLimit() {
     return this.config.scoreLimit;
   }
 
+  public void setScoreLimit(int scoreLimit) {
+    if (hasMercyRule()) {
+      mercyRule.setScoreLimit(scoreLimit);
+      return;
+    }
+    this.config.scoreLimit = scoreLimit;
```

Comment:
The fact that config isn't immutable is an issue in itself, it shoud always have been, but that aside, this change will mantain itself until the server is restarted, that's bad. 

The "config" represents the values in the XML, they must NOT be modified by pgm during runtime, except when it's to re-parse the XML and generate a new definition (eg: if you modify the xml, or upload a different version of the map), the match-scoped module should never modify the map-scoped value.